client/src/components/categoryButtons/CategoryControl.jsx

```diff
@@ -1,38 +1,38 @@
 import React, { useState, useEffect } from 'react';
 import ButtonsDisplay from './ButtonsDisplay.jsx'
 
-export default function categoryControl(props) {
-  const [isLongList, setIsLongList] = useState(false);
-  useEffect(() => {
-    if (props.categories.length >= 5) {
-      setIsLongList(true);
-    }
-  }, []);
+  export default function categoryControl(props) {
```

Comment:
a good practice when you have 3 or less props is to destructure them either inside the parentheses on line 4 or immedately on line 5.

```
export default function categoryControl({ categories }) {
```

or

```
const { categories } = props;
```